app/models/assignment.rb

```diff
@@ -82,7 +82,7 @@ def visible_status
     end
   end
 
-  def persist_submission!(submission)
+  def save_submission!(submission)
```

Comment:
I renamed `persist_submission!` and `setup_query_assignment!` for consistency, since they actually talked about the same thing as `Submission#save_submission!`